testenv/osccluster_controller_test.go

```diff
@@ -707,14 +707,9 @@ var _ = Describe("Outscale Cluster Reconciler", func() {
 			if !ok {
 				osc_region = "eu-west-2"
 			}
-			osc_subregion, ok := os.LookupEnv("OSC_SUBREGION_NAME")
```

Comment:
Why do you delete it ?
Test can be on subreigon which are not eu-west-2a .
It will be for example for our ci on us-east-2a.
And also it is possible that the zone are b instead of a